digicampipe/io/containers.py

```diff
@@ -44,6 +47,38 @@
            'DataContainer']
 
 
+class CameraEventType(IntFlag):
+    '''
+    I do not know where this comes from, but I found it here:
+    https://github.com/cta-sst-1m/digicampipe/issues/244
+
+    ALG0       (0x1)     /* algorithm 0 trigger - PATCH7 */
+    ALG1       (0x2)     /* algorithm 1 trigger - PATCH19 */
+    ALG2       (0x4)     /* algorithm 2 trigger - MUON */
+    INTRNL     (0x8)     /* internal or external  */
+    EXTMSTR    (0x10)    /* unused (0) / external (on master only)  */
+    BIT5       (0x20)    /* unused (0)  */
+    BIT6       (0x40)    /* unused (0)  */
+    CONT       (0x80)    /* continuous readout marker  */
+    CSP_MUON   (0x10000) /* camera server detected muon*/
+    CSP_HILLAS (0x20000) /* camera server computed Hillas parametrs */
+
+    I removed the EVTTYPE, since this is clear from the class name
+    I removed DIGICAM since this is clear from the project.
+    '''
+    UNKNOWN = 0x0
```

Comment:
I wonder if we should use the same naming as done in other code.  I would prefer that names are words. e.g. : 

ALG0 --> ALGORITHM0
INTRNL --> INTERNAL
EXTMSTR --> ??? I don't even know what it is
CONT --> ???
CSP_MUON -- ??? is it some king of cameraserverflag?

Reply:
I agree, but if *we* as SST1M digicam people now decide we want to rename these. Do we ask the camera server people to also rename? Or do we create another wall of mental mapping?